src/bindings/python/fluxacct/accounting/Makefile.am

```diff
@@ -6,7 +6,7 @@ acctpy_PYTHON = \
 	print_hierarchy.py
 
 dist_fluxcmd_SCRIPTS = \
-	flux-account.py
+	../../../../cmd/flux-account.py
```

Comment:
I think it might make more sense to delete these two lines from this `Makefile.am` and add them to the `Makefile.am` in `src/cmd`.